multiqc/utils/config.py

```diff
@@ -185,3 +186,14 @@ def load_sample_names(snames_file):
     except (IOError, AttributeError) as e:
         logger.error("Error loading sample names file: {}".format(e))
     logger.debug("Found {} sample renaming patterns".format(len(sample_names_rename_buttons)))
+
+def update_dict(d, u):
```

Comment:
Anyway it doesn't work in `util_functions` because `config` is imported there and it would cause an import loop